emu/src/cpu/instruction.rs

```diff
@@ -4,34 +4,66 @@ use logger::log;
 
 use crate::bitwise::Bits;
 
+use super::condition::Condition;
+
 #[derive(Debug, PartialEq, Eq)]
 pub enum ArmModeInstruction {
```

Comment:
Is there the possiblity to have `Cond` for every option without the need to specify it everytime?

Reply:
Do you mean for every variant of enum? Nope

Reply:
Or with a macro but...